static/js/composebox_typeahead.js

```diff
@@ -185,7 +184,6 @@ export function handle_enter(textarea, e) {
         textarea.caret("\n");
         compose_ui.autosize_textarea(textarea);
         e.preventDefault();
-        return;
```

Comment:
we dont need `return` further here in this function

Reply:
I think it'd be most readable to reorder this function.

Reply:
ok will be reordering it